### PR TITLE
DAM-7259 Update labels

### PR DESCRIPTION
**Pull request by bbedigi**

### CODE REVIEW

File: DC/5.10.0/mail_templates/assetsharedbyemail.dcl [hunk 1]
```diff
@@ -2,19 +2,19 @@ resource mail_template assetsharedbyemail {
     portal_name = ''
     language_id = resource.language.english.id
     template_name = 'asset-shared-by-email'
-    subject = "{{sender.name}} has invited you to join a shared asset in Digizuite' Media Manager"
+    subject = "{{sender.name}} has invited you to download a shared asset from Digizuite' Media Manager"
```
Comment (Poiar) [inline DC/5.10.0/mail_templates/assetsharedbyemail.dcl:5]:
```suggestion
    subject = "{{sender.name}} has invited you to download a shared asset from Digizuite&#8482; Media Manager"
```

File: DC/5.10.0/mail_templates/assetsharedbyemail.dcl [hunk 2]
```diff
@@ -2,19 +2,19 @@ resource mail_template assetsharedbyemail {
...
 {{include \'html-header-end\'}}
 
-<span class="preheader">{{sender.name  | html.escape}} has invited you to join a shared asset in Digizuite&#8482; Media Manager</span>
+<span class="preheader">{{sender.name  | html.escape}} has invited you to download a shared asset from Digizuite&#8482; Media Manager</span>
 
 {{include \'standard-header\'}}
 
 <h1>Hi {{receiver.name | html.escape}},</h1>
-<p>{{sender.name | html.escape}} ({{sender.email_address | html.escape}}) has invited you to join a shared asset in Digizuite&#8482; Media Manager. Enjoy!
+<p>{{sender.name | html.escape}} ({{sender.email_address | html.escape}}) has invited you to download a shared asset from Digizuite&#8482; Media Manager. Enjoy!
```
Comment (Poiar) [inline DC/5.10.0/mail_templates/assetsharedbyemail.dcl:17]: That "Enjoy!" is ominous

File: DC/5.9.0/mail_templates/assetsharedbyemail.dcl [hunk 3]
```diff
@@ -2,19 +2,19 @@ resource mail_template assetsharedbyemail {
     portal_name = ''
     language_id = resource.language.english.id
     template_name = 'asset-shared-by-email'
-    subject = "{{sender.name}} has invited you to join a shared asset in Digizuite' Media Manager"
+    subject = "{{sender.name}} has invited you to download a shared asset from Digizuite' Media Manager"
```
Comment (Poiar) [inline DC/5.9.0/mail_templates/assetsharedbyemail.dcl:5]:
```suggestion
    subject = "{{sender.name}} has invited you to download a shared asset from Digizuite&#8482; Media Manager"
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: DC/5.10.0/mail_templates/assetsharedbyemail.dcl (hunk 2 above)
Comment (bbedigi) [reply DC/5.10.0/mail_templates/assetsharedbyemail.dcl:17]: Ha. yes but this will be customised per client anyway i would think

**Review by Poiar — APPROVED** [review]

**Pull request merged**